website/docs/r/compute_cluster.html.markdown

```diff
@@ -500,6 +506,9 @@ resource "vsphere_compute_cluster" "compute_cluster" {
   vsan_verbose_mode_enabled = true
   vsan_network_diagnostic_mode_enabled = true
   vsan_unmap_enabled = true
+  vsan_remote_datastore_ids   = [data.vsphere_datastore.datastore.id]
```

Comment:
This configuration example would fail as HCI Mesh and DIT [are not compatible](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vsan.doc/GUID-9113BBD6-5428-4287-9F61-C8C3EE51E07E.html).